.github/workflows/run_fvt_tests.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
         - uses: actions/checkout@v1
         - uses: erlef/setup-beam@v1
           with:
-            otp-version: "23.3.4.9"
+            otp-version: "23.3.4.17"
```

Comment:
Here and below: shouldn't it be 18?

Reply:
Wasn't sure if 23.3.4.18 was available for this action when I was fixing it, but yeah, most likely it should be fine to put 18 here as well.
For the sake of the original issue - 23.3.4.15 is enough to address [CVE-2022-37026](https://cve.report/CVE-2022-37026).

Reply:
When I update the emqx-builder repo, there only was 23.3.4.17.
If we want to use the newest version of OTP, we need update the emqx-builder again, do we need ?

Reply:
17 should be ok I thinj